packages/playground-ui/src/ds/components/Shimmer/shimmer.css

```
@@ -1,18 +1,21 @@
 /* Sweeping highlight for text that is still being produced.
  * The gradient period is a fixed pixel width and the animation travels exactly
  * one period, so the loop never jumps and a short label sweeps at the same
- * speed as a long one. */
+ * speed as a long one. The sweep is relative to the inherited color, so muted
+ * text stays muted and only brightens as the highlight passes through it. */
 .shimmer-text {
-  --shimmer-period: 180px;
+  --shimmer-period: 120px;
+  --shimmer-dim: color-mix(in oklab, currentColor 55%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)